control/plugin/plugin.go

```diff
@@ -91,7 +91,7 @@ var (
 		"collector",
 		"processor",
 		"publisher",
-		"streamCollector",
+		"streaming collector",
```

Comment:
@jcooklin, I prepared PR with that change: -> https://github.com/jcooklin/snap/pull/2 - use it or not, up to you